.mailmap

```diff
@@ -1149,6 +1149,9 @@ Tiffany Zhu <bubble.wubble.303@gmail.com>
 Tim Gates <tim.gates@iress.com>
 Tim Lahey <tim.lahey@gmail.com>
 Tim Swast <tswast@gmail.com>
+Timo Stienstra <97806294+TJStienstra@users.noreply.github.com>
+Timo Stienstra <T.J.Stienstra@student.tudelft.nl>
+Timo Stienstra <timostienstra00@gmail.com>
```

Comment:
The purpose of this file is to show when commits with different metadata belong to the same person and to say what name and email address should be used for recording that person's identity in the AUTHORS file. Here you are listing three separate authors. If you run the `mailmap_check.py` script you will see:
```console
$ bin/mailmap_check.py 
No changes needed in .mailmap

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Andrii Oriekhov <andriyorehov@gmail.com>
	Arie Bovenberg <a.c.bovenberg@gmail.com>
	Carson McManus <carson.mcmanus1@gmail.com>
	Gaurav Jain <gjain369@gmail.com>
	Jack Schmidt <1107865+jackschmidt@users.noreply.github.com>
	Luis Talavera <luisfertalavera15@gmail.com>
	Shivam Sagar <technoshivam12@gmail.com>
	Siddhant Jain <siddhantashoknagar@gmail.com>
	Sumit Kumar <mr.sumitkrr@gmail.com>
	Timo Stienstra <T.J.Stienstra@student.tudelft.nl>
	Timo Stienstra <timostienstra00@gmail.com>
	Tirthankar Mazumder <63574588+wermos@users.noreply.github.com>
```
This is showing that it things that there are 2 different people called Timo Stienstra. The third entry with the no reply email address only appears in CI because a temporary merge commit is created by GitHub.

Instead what you should add here (in .mailmap) is something like these two lines:
```
Timo Stienstra <timostienstra00@gmail.com> Timo Stienstra <T.J.Stienstra@student.tudelft.nl>
Timo Stienstra <timostienstra00@gmail.com> Timo Stienstra <97806294+TJStienstra@users.noreply.github.com>
```
These lines associate the other email addresses with the same person so that then only a single person will be added to the AUTHORS file as `￼Timo Stienstra <timostienstra00@gmail.com>`.

Reply:
Thank you, I assume I should also add:
```Timo Stienstra <timostienstra00@gmail.com> Timo Stienstra <timostienstra00@gmail.com>```
Since I just switched to that mail (the other one I use on my university account)

Reply:
You don't need to map the same email address to itself.

Reply:
You should run the `mailmap_check.py` script yourself before pushing:
```console
$ bin/mailmap_check.py 
This author is not included in the .mailmap file:
TJStienstra <T.J.Stienstra@student.tudelft.nl>
Ambiguous names in .mailmap
This email address appears for multiple entries:
Person: TJStienstra <timostienstra00@gmail.com>
Mailmap entries:
Timo Stienstra <timostienstra00@gmail.com> Timo Stienstra <97806294+TJStienstra@users.noreply.github.com>
Timo Stienstra <timostienstra00@gmail.com> Timo Stienstra <T.J.Stienstra@student.tudelft.nl>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.

Lines should be added to .mailmap to indicate the correct name and
email aliases for all commits.

For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Andrii Oriekhov <andriyorehov@gmail.com>
	Arie Bovenberg <a.c.bovenberg@gmail.com>
	Carson McManus <carson.mcmanus1@gmail.com>
	Gaurav Jain <gjain369@gmail.com>
	Jack Schmidt <1107865+jackschmidt@users.noreply.github.com>
	Luis Talavera <luisfertalavera15@gmail.com>
	Shivam Sagar <technoshivam12@gmail.com>
	Siddhant Jain <siddhantashoknagar@gmail.com>
	Sumit Kumar <mr.sumitkrr@gmail.com>
	Tirthankar Mazumder <63574588+wermos@users.noreply.github.com>
	TJStienstra <T.J.Stienstra@student.tudelft.nl>
	TJStienstra <timostienstra00@gmail.com>
```
The problem is that your name (as recorded in the commits) is TJStienstra rather than Timo Stienstra:
https://github.com/sympy/sympy/commit/d76c7ac376daaa46cf376ad7743532c0840cf7dd.patch

You need to associate the name and email address that is in the commits with the name and email address that should be recorded in the .mailmap file.

Reply:
Oh sorry, I only read the lowest part of the output and thought that it was fixed. However if I change it to:
```
Timo Stienstra <timostienstra00@gmail.com> TJStienstra <97806294+TJStienstra@users.noreply.github.com>
Timo Stienstra <timostienstra00@gmail.com> TJStienstra <T.J.Stienstra@student.tudelft.nl>
```
Which should be fine, but it still gives the error:
```
←This email address appears for multiple entries:
Person: TJStienstra <timostienstra00@gmail.com>
Mailmap entries:
Timo Stienstra <timostienstra00@gmail.com> TJStienstra <97806294+TJStienstra@users.noreply.github.com>
Timo Stienstra <timostienstra00@gmail.com> TJStienstra <T.J.Stienstra@student.tudelft.nl>
```
Even when removing the noreply.github alias.

Reply:
It isn't necessary to delete the PR if you want to do that. You can just set up your name and email address with `git config`, redo the commits and then force-push to the same branch.

Reply:
Okay (I already setup the correct stuff with `git config`), so now I can create a local branch with the same name `FixVelocityCalculation` and do the correct changes and `push --force` it upstream, overwriting the current branch?